services/auth/src/main/resources/application-dev.properties

```diff
@@ -0,0 +1,18 @@
+### Used when starting quarkus app in development mode, e.g. mvn quarkus:dev
+
+app.id=auth
+
+# amqp APU server
+hono.auth.amqp.bindAddress=0.0.0.0
+hono.auth.amqp.port=${fixed.auth.amqps.port:5673}
+hono.auth.amqp.keyPath=../../demo-certs/certs/${app.id}-server-key.pem
```

Comment:
In most modules we already unpack the demo certs into the target folder. Not sure if that is possible, but if we were able to load the files from the class path, we could refrain from making assumptions regarding the current working directory and traversing the file system up and down ...

Reply:
Maybe we can take a look at that later ...